### DISCUSSION TITLE
YouTube No Longer Allows Login in Qutebrowser

### DISCUSSION THREAD

**Discussion by baiguai** [qutebrowser/qutebrowser#8128] [General]
I've been using QB to check YouTube stats every day for about a year. As of last night, YT has stopped allowing logging in - using QB.

```
This browser or app may not be secure. [Learn more](https://support.google.com/accounts/answer/7675428?hl=en)
Try using a different browser. If you’re already using a supported browser, you can try again to sign in.
```

I'm not sure if anything can be done about this, but thought it would be good to bring attention to this. Very frustrating update on YT's part.

**Comment by maccagnan** [comment]
This might or might not help, but I experienced similar issues from time to time, and it wasn't a QB problem but rather my IP modem configuration that needed a resetting.

**Comment by mschilli87** [comment]
Just out of curiousity, everytime somebody ask here for the tiny team of people working for free on a browser to implement some workaround for a webpage deliberately broken by a big corporation with the sole purpose to force more users into their ecosystem to collect data and make money, does anyone ever also open a bug report with the big company paying thousands of people to run that broken website?
I'd love to see the Youtube/Google bug report about this.

**Comment by kd4e** [comment]
Sine qutebrowser uses the Chromium-Blink browser engine, has a tiny user-base, and since google/alphabet owns Chrome (and funds Chromium development) this seems less about monopoly vs competition and might be a security concern? I'm reading that qutebrowser uses a 2020 or 2022 version of the browser engine - is that correct?

**Reply by mschilli87** [reply]
Qt is backporting security patches. Also, how would me using an outdated browser be a security concern for Youtube? And how does blocking me from their (hopefully safe) site increase my security? And evil website exploiting my old browser would still happily take my requests. And the size of the user base does not matter. This is not agains qutebrowser. This is against everything that is not Chrome or pays Google in some other way. This form of gatekeeping keeps alternative engines small. And given that it is a crazy task to write a web engine from scratch, this strategy has worked to create and maintain the monopoly for a long time. Which gives them the power to decide how the web wil develop. How is ad delivery done? Which DRMs are used? What image formats gain traction? Support by Google Chrome (or the lack therof) determaines all those aspects.

**Reply by The-Compiler** [reply]
> I'm reading that qutebrowser uses a 2020 or 2022 version of the browser engine

Where are you reading that? On Linux, it uses Chromium from whatever Qt(WebEngine) version you have installed, it doesn't ship with Qt. So this doesn't make any sense at all. If it uses an outdated Chromium, it's because your system has an outdated Qt - but that's not on qutebrowser.

I'm currently using it with Qt 6.6.2, which is based on Chromium 112 (April 2023), with security fixes backported from 121 (February 2024). Qt 6.6.3 is already released, so as soon as that makes it into Arch, I'll have security fixes backported from Chromium 122 (March 2024). Then in around a week, Qt 6.7 will be released, based on Chromium 118 (October 2023), again with security fixes backported from Chromium 122.
